lib/season_stat_team.rb

```diff
@@ -0,0 +1,80 @@
+require_relative 'game_collection'
+require_relative 'game_team_collection'
+require_relative 'team_collection'
+
+class SeasonStatTeam
```

Comment:
I have a pretty good idea of how i might refactor this, all of these classes are using very similar methods so I think it might be a good use case for inheritance with the season_stat class

Reply:
These already look great. For a stand alone class I dont think these need refactoring, but once we put everything together we'll probably see repeats